cli/internal/runsummary/run_summary.go

```diff
@@ -68,6 +71,7 @@ type RunSummary struct {
 	FrameworkInference bool               `json:"frameworkInference"`
 	ExecutionSummary   *executionSummary  `json:"execution,omitempty"`
 	Tasks              []*TaskSummary     `json:"tasks"`
+	User               string             `json:"user"`
```

Comment:
Do we want `"user": ""` in the JSON?

Reply:
This only shows up in dry run json and run summary. I think empty string is ok and will make issues more obvious